test/metabase/api/notify_test.clj

```diff
@@ -38,11 +60,10 @@
             post       (fn post-api
                          ([payload] (post-api payload 200))
                          ([payload expected-code]
-                          (mt/client :post expected-code (format "notify/db/%d" (u/the-id (mt/db)))
-                                     {:request-options
-                                      {:headers {"X-METABASE-APIKEY" "test-api-key"
-                                                 "Content-Type"      "application/json"}}}
-                                     payload)))]
+                          (mt/with-temporary-setting-values [api-key "test-api-key"]
```

Comment:
This (and the one above) lets the test work correctly when run from the REPL. See [Cam's comment here](https://metaboat.slack.com/archives/CKZEMT1MJ/p1664300567452529?thread_ts=1664294388.338909&cid=CKZEMT1MJ)